### PR TITLE
AD-13685: Dry run and provide the Report

### PR DESCRIPTION
**Pull request by Rakshitha-Sheshagiri**

### CODE REVIEW

**Comment by Rakshitha-Sheshagiri** [comment]
[email_migration_1666782488492.csv](https://github.com/Bluescape/bluescape-cli/files/9868912/email_migration_1666782488492.csv)

**Review by vinothkannanmani — APPROVED** [review]
LGTM

File: src/commands/emailmigration/execute.ts [hunk 1]
```diff
@@ -16,11 +18,11 @@ import { Builder, Handler } from '../user/get.types';
 import { ApplicationRole, Roles } from '../user/role.types';
 import { askOrganizationId } from './ask-migration-information';
 
-export const command = 'execute';
-export const desc = 'Execute migration of member emails';
+export const command = 'dry-run';
```
Comment (HMageswari) [inline src/commands/emailmigration/execute.ts:21]: I only see the command changes, but the file name remains `execute`. This file name needs to be changed. Have you test this?

File: src/commands/emailmigration/execute.ts [hunk 2]
```diff
@@ -16,11 +18,11 @@ import { Builder, Handler } from '../user/get.types';
...
+export const desc = 'Dry run migration of member emails';
 
 export const builder: Builder = (yargs) =>
-  yargs.example([['$0 emailmigration execute --mapping-csv=xx.csv']]);
+  yargs.example([['$0 emailmigration dry-run --mapping-csv=xx.csv']]);
```
Comment (HMageswari) [inline src/commands/emailmigration/execute.ts:25]: This completely removes the execute command.

**Pull request closed without merging**